contracts/index.md

```diff
@@ -28,80 +29,103 @@ Set an Locator on the Index.
 
 ```java
 function setLocator(
-  address _signaller,
-  uint256 _score,
-  bytes32 _data
+  address identifier,
+  uint256 score,
+  bytes32 locator
 ) external onlyOwner
 ```
 
-| Param        | Type      | Description                                  |
-| :----------- | :-------- | :------------------------------------------- |
-| `_score`     | `uint256` | Score for placement in the list.             |
-| `_signaller` | `address` | The account or contract setting the Locator. |
-| `_data`      | `bytes32` | Arbitrary data.                              |
+| Param            | Type      | Description                                         |
+| :--------------- | :-------- | :-------------------------------------------------- |
+| `identifier`     | `address` | On-chain address identifying the owner of a locator.|
+| `score`          | `uint256` | Score for the locator being set.                    |
+| `locator`        | `bytes32` | Locator.                                            |
 
 A successful `setLocator` emits a `SetLocator` event.
 
 ```java
 event SetLocator(
+  address indexed identifier,
   uint256 score,
-  address indexed signaller,
-  bytes32 indexed data
+  bytes32 indexed locator
 );
 ```
 
 ---
 
-| Revert Reason         | Scenario                                        |
-| :-------------------- | :---------------------------------------------- |
-| `LOCATOR_ALREADY_SET` | A Locator by the same signaller is already set. |
+| Revert Reason         | Scenario                                         |
+| :-------------------- | :----------------------------------------------  |
+| `ENTRY_ALREADY_EXISTS` | A Locator by the same signaller is already set. |
```

Comment:
`signaller` is no longer terminology we use. and `Locator` is now `Entry`
-> for example
`This address already has an Entry on the Index. `